model/rpc/common.yaml

```diff
@@ -0,0 +1,33 @@
+groups:
+  - id: attributes.metrics.rpc.server
+    type: attribute_group
+    brief: "Describes RPC metric attributes."
+    attributes:
+      - ref: rpc.system
+        requirement_level: required
+      - ref: rpc.service
+        note: ""
+      - ref: rpc.method
+        note: ""
```

Comment:
why do we need to remove attribute notes on all method and service? if the notes are not right, let's create an issue to correct them, but overriding them everywhere does not seem necessary or right